scripts/release.py

```diff
@@ -46,7 +47,7 @@ def main():
     jar = build_binary()
 
     # release
-    release_command = ['hub', 'release', 'create', '-a', jar, '-m', title, tag_name]
```

Comment:
This can be improved with f-string.

 https://realpython.com/python-f-strings/